tidb-configuration-file.md

```diff
@@ -181,7 +181,7 @@ The TiDB configuration file supports more options than command-line parameters.
 ### `deprecate-integer-display-length`
 
 - Deprecates the display width for integer types when this configuration item is set to `true`.
-- Default value: `false`
+- Default value: `true`
```

Comment:
Question: For a cluster upgraded from v8.4 or earlier versions, will the default value remain unchanged (still be `false`) after the upgrade?